packages/react-core/src/components/Select/Select.tsx

```diff
@@ -353,36 +353,44 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
           return React.cloneElement(group, {
             titleId: group.props.label && group.props.label.replace(/\W/g, '-'),
             children: React.Children.map(group.props.children, (child: React.ReactElement) =>
-              React.cloneElement(child as React.ReactElement, {
-                isFocused:
-                  activeElement &&
-                  (activeElement.id === (child as React.ReactElement).props.id ||
-                    (this.props.isCreatable &&
-                      typeaheadActiveChild.innerText === `{createText} "${(child as React.ReactElement).props.value}"`))
-              })
+              child.type === Divider
+                ? child
+                : React.cloneElement(child as React.ReactElement, {
+                    isFocused:
+                      activeElement &&
+                      (activeElement.id === (child as React.ReactElement).props.id ||
+                        (this.props.isCreatable &&
+                          typeaheadActiveChild.innerText ===
+                            `{createText} "${(child as React.ReactElement).props.value}"`))
+                  })
             )
           });
         } else {
-          return React.cloneElement(group, {
-            isFocused:
-              typeaheadActiveChild &&
-              (typeaheadActiveChild.id === (group as React.ReactElement).props.id ||
-                (this.props.isCreatable &&
-                  typeaheadActiveChild.innerText === `{createText} "${(group as React.ReactElement).props.value}"`))
-          });
+          return group.type === Divider
+            ? group
+            : React.cloneElement(group, {
+                isFocused:
+                  typeaheadActiveChild &&
+                  (typeaheadActiveChild.id === (group as React.ReactElement).props.id ||
+                    (this.props.isCreatable &&
+                      typeaheadActiveChild.innerText === `{createText} "${(group as React.ReactElement).props.value}"`))
+              });
         }
       });
     }
 
-    return typeaheadChildren.map((child: React.ReactNode) =>
-      React.cloneElement(child as React.ReactElement, {
-        isFocused:
-          typeaheadActiveChild &&
-          (typeaheadActiveChild.innerText === (child as React.ReactElement).props.value.toString() ||
-            (this.props.isCreatable &&
-              typeaheadActiveChild.innerText === `{createText} "${(child as React.ReactElement).props.value}"`))
-      })
-    );
+    return typeaheadChildren.map((child: React.ReactNode) => {
+      const childElement = child as any;
+      return childElement.type.displayName === 'Divider'
```

Comment:
Can you cast to a ReactNode instead and then check `childElement.type` like you do for all the other component logic?Like this: `(childElement as React.ReactNode).type === Divider`?

Reply:
That is what I tried first.  It did not like that.  Elsewhere the child is a ReactElement so I can check the `type`.